engine/access/rpc/backend/backend_stream_transactions.go

```diff
@@ -30,55 +33,132 @@ type backendSubscribeTransactions struct {
 
 	subscriptionHandler *subscription.SubscriptionHandler
 	blockTracker        subscription.BlockTracker
+	sendTransaction     sendTransaction
 }
 
-// TransactionSubscriptionMetadata holds data representing the status state for each transaction subscription.
-type TransactionSubscriptionMetadata struct {
+// transactionSubscriptionMetadata holds data representing the status state for each transaction subscription.
+type transactionSubscriptionMetadata struct {
 	*access.TransactionResult
 	txReferenceBlockID   flow.Identifier
 	blockWithTx          *flow.Header
 	txExecuted           bool
 	eventEncodingVersion entities.EventEncodingVersion
+	shouldTriggerPending bool
 }
 
-// SubscribeTransactionStatuses subscribes to transaction status changes starting from the transaction reference block ID.
-// If invalid tx parameters will be supplied SubscribeTransactionStatuses will return a failed subscription.
-func (b *backendSubscribeTransactions) SubscribeTransactionStatuses(
+// SendAndSubscribeTransactionStatuses sends a transaction and subscribes to its status updates.
+// It starts monitoring the status from the transaction's reference block ID.
+// If the transaction cannot be sent or an error occurs during subscription creation, a failed subscription is returned.
+func (b *backendSubscribeTransactions) SendAndSubscribeTransactionStatuses(
 	ctx context.Context,
 	tx *flow.TransactionBody,
 	requiredEventEncodingVersion entities.EventEncodingVersion,
 ) subscription.Subscription {
-	nextHeight, err := b.blockTracker.GetStartHeightFromBlockID(tx.ReferenceBlockID)
+	if err := b.sendTransaction(ctx, tx); err != nil {
+		b.log.Error().Err(err).Str("tx_id", tx.ID().String()).Msg("failed to send transaction")
```

Comment:
avoid logging errors for api client issues since this can cause extremely noisy error level logging for larger operators.
```suggestion
		b.log.Debug().Err(err).Str("tx_id", tx.ID().String()).Msg("failed to send transaction")
```